### PR TITLE
LA County case studies page layout

### PR DESCRIPTION
**Pull request by jakewheeler**
This PR adds the first case study page layout.

### CODE REVIEW

File: src/app/case-studies/page.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export default function CaseStudies() {
                 data pipeline that automatically processes and enriches eCR data
                 to improve downstream data analysis and case investigation.
               </Text>
-              <LinkButton variant="primary" href="/">
+              <LinkButton variant="primary" href="/case-studies/la-county">
```
Comment (jakewheeler) [inline src/app/case-studies/page.tsx:32]: This route is a placeholder that may change later.

File: src/app/case-studies/la-county/page.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
+import Link from 'next/link';
+import { Link as ExternalLink } from '@trussworks/react-uswds';
+import './styles.scss';
+
+export default function LaCountyCaseStudy() {
```
Comment (jakewheeler) [inline src/app/case-studies/la-county/page.tsx:5]: The case study pages all have a very similar layout. When working on the next case study page, I'll start looking into breaking these sections out into layout components.

**Review by nickbristow — CHANGES_REQUESTED** [review]
The column of text seems off center. 
image

**Comment by jakewheeler** [comment]
> The column of text seems off center. 

@nickbristow great catch, thanks! I believe I've resolved this but let me know if anything still seems off at all.

**Review by nickbristow — APPROVED** [review]

**Pull request merged**